.gitignore

```diff
@@ -13,3 +13,7 @@ yarn.lock
 #build folder
 /build/
 
+
+#Text files -- Personal notes
+*.txt
```

Comment:
might be a good idea to specified which file in a case that we would like to use txt, but we can take care about it when it is needed